pycobertura/reporters.py

```diff
@@ -443,3 +443,49 @@ def generate(self):
                     )
 
         return template.render(**render_kwargs)
+
+
+class GitHubAnnotationReporter(Reporter):
+    def __init__(
+        self,
+        cobertura,
+        ignore_regex=None,
+        annotation_level: str = "error",
```

Comment:
Will an `annotation_level` of "error" fail a pull request?

Reply:
No. It's more like a `logger.error("not covered")`. The overall workflow status still depends on the status of each steps' exit code.

<img width="1368" alt="Screenshot 2023-04-19 at 1 44 29 PM" src="https://user-images.githubusercontent.com/22942154/232978227-795626c1-2683-492f-82a3-2fcb114f48d7.png">

Reply:
https://github.com/aconrad/pycobertura/pull/164/commits/1ddf370ecd4693eab715410f22b8f749b7325438

Reply:
screenshot updated in files and PR scrrenshot